### PR TITLE
Remove old (duplicate) worker tests

### PR DESCRIPTION
**Pull request by michaljurecko**
Jira: https://keboola.atlassian.net/browse/PSGO-100

Changes:
- Removed old, duplicate, worker tests, ... they have been replaced by new test in https://github.com/keboola/keboola-as-code/pull/1225

### CODE REVIEW

File: internal/pkg/service/buffer/worker/service/conditions_test.go [hunk 1]
```diff
@@ -1,346 +0,0 @@
...
-	"github.com/keboola/keboola-as-code/internal/pkg/service/common/dependencies"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/etcdhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/strhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/testhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/testproject"
-)
-
-func TestConditionsChecker(t *testing.T) {
-	t.Parallel()
```
Comment (michaljurecko) [inline internal/pkg/service/buffer/worker/service/conditions_test.go:31]: Conditions su po novom E2E otestovane tu:

Upload:
- https://github.com/keboola/keboola-as-code/blob/718568f24aaf898fb528a35b4aba9dfadd71741b/test/buffer/worker/001_import_records_test.go#L7-L8
- https://github.com/keboola/keboola-as-code/blob/718568f24aaf898fb528a35b4aba9dfadd71741b/test/buffer/worker/002_slice_upload_test.go#L21-L22


Import:
https://github.com/keboola/keboola-as-code/blob/718568f24aaf898fb528a35b4aba9dfadd71741b/test/buffer/worker/003_file_import_test.go#L26-L27

Comment (michaljurecko) [reply internal/pkg/service/buffer/worker/service/conditions_test.go:31]: Tam ^^^ sa to testuje nad `rows count`, ale z E2E hladiska to je jedno.

Unit testy pre vsetky druhy conditions mame tu:
https://github.com/keboola/keboola-as-code/blob/718568f24aaf898fb528a35b4aba9dfadd71741b/internal/pkg/service/buffer/store/model/conditions_test.go#L56

File: internal/pkg/service/buffer/worker/service/file_test.go [hunk 2]
```diff
@@ -1,629 +0,0 @@
...
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/worker/service"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/common/dependencies"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/etcdhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/strhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/testhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/testproject"
-)
-
-func TestUploadAndImportE2E(t *testing.T) {
```
Comment (michaljurecko) [inline internal/pkg/service/buffer/worker/service/file_test.go:31]: To je vsetko pokryte uz tu:
https://github.com/keboola/keboola-as-code/blob/718568f24aaf898fb528a35b4aba9dfadd71741b/test/buffer/worker/worker_test.go#L25-L31

File: internal/pkg/service/buffer/worker/service/slice_close_test.go [hunk 3]
```diff
@@ -1,333 +0,0 @@
...
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/worker/service"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/common/dependencies"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/etcdhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/strhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/testhelper"
-)
-
-// TestSliceCloseTask - the worker closes the slice only after it is not used by any API node.
-func TestSliceCloseTask(t *testing.T) {
```
Comment (michaljurecko) [inline internal/pkg/service/buffer/worker/service/slice_close_test.go:25]: Opat, je to pokryte:
https://github.com/keboola/keboola-as-code/blob/718568f24aaf898fb528a35b4aba9dfadd71741b/test/buffer/worker/worker_test.go#L25-L31

Konkretne tu sa testuje, ze najpr musia vsetky API nodes oznami, ze sa prepli na novu slices, az tak sa odblokuje `slice.close` task:
https://github.com/keboola/keboola-as-code/blob/718568f24aaf898fb528a35b4aba9dfadd71741b/test/buffer/worker/002_slice_upload_test.go#L41-L53

File: internal/pkg/service/buffer/worker/service/slice_upload_test.go [hunk 4]
```diff
@@ -1,734 +0,0 @@
...
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/etcdhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/strhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/testhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/testproject"
-)
-
-// TestSliceUploadTask - there are 2 slices, one is empty.
-// Both sliced are closed, but only the non-empty slice is uploaded to file storage.
-func TestSliceUploadTask(t *testing.T) {
```
Comment (michaljurecko) [inline internal/pkg/service/buffer/worker/service/slice_upload_test.go:31]: Je to pokryte:
https://github.com/keboola/keboola-as-code/blob/718568f24aaf898fb528a35b4aba9dfadd71741b/test/buffer/worker/worker_test.go#L25-L31

Obsah suboru nemusi kontrolovat, ked kontrolujeme obsah vyslednej tabulky po importe, tu:
https://github.com/keboola/keboola-as-code/blob/718568f24aaf898fb528a35b4aba9dfadd71741b/test/buffer/worker/003_file_import_test.go#L69-L99

**Review by michaljurecko — COMMENTED** [review]
V tomto PR mazem stare testy, ktore nahradili nove testy.

V novych testoch sa testuje zapis z jedneho receiver do 2 exports, takze je to tam este viac otestovane, ako to bolo v tychto ciastocnych testoch.

**Comment by michaljurecko** [comment]
(`slice_retry_test.go` zatial zostal, lebo este zan nie je nahrada)

**Review by JakubMatejka — APPROVED** [review]

**Pull request merged**